std_msgs/Header header
float64 Kp
float64 Ki
float64 Kd
float64 error
float64 error_dot
float64 error_integral
float64 control_input
bool clegg_triggered = False
bool windup_triggered = False